recipes-bsp/grub/grub-xen-pvh_2.04.bb

```diff
@@ -0,0 +1,4 @@
+require grub-xen.inc
+GRUBPLATFORM = "xen_pvh"
```

Comment:
Is `xen_pvh` turned on by default in grub2 now? The xen instructions here from 4.12 make a note of compiling it using the `--with-platform=xen_pvh` flag. https://wiki.xenproject.org/wiki/Xen_4.12_RC_test_instructions.

Just making sure we don't need to do anything to the base grub2 recipe

Reply:
GRUBPLATFORM is the variable that is passed to `--with-platform`:

https://github.com/openembedded/openembedded-core/blob/f45ddfbf007de858327eef0ffefd5840ef4c69b8/meta/recipes-bsp/grub/grub2.inc#L77